### PR TITLE
PRO-1147 3/language

### PR DESCRIPTION
**Pull request by stuartromanek**
https://linear.app/apostrophecms/issue/PRO-1147

### CODE REVIEW

File: modules/@apostrophecms/admin-bar/ui/apos/components/TheAposContextTitle.vue [hunk 1]
```diff
@@ -131,10 +131,6 @@ export default {
   &__document {
     margin-top: 3.5px;
   }
-
-  & /deep/ .apos-indicator {
-    margin-top: 1px;
-  }
```
Comment (stuartromanek) [inline modules/@apostrophecms/admin-bar/ui/apos/components/TheAposContextTitle.vue:137]: unrelated fiddling

File: modules/@apostrophecms/doc-type/index.js [hunk 2]
```diff
@@ -6,7 +6,8 @@ module.exports = {
     contextBar: true,
     editRole: 'contributor',
     publishRole: 'editor',
-    viewRole: false
+    viewRole: false,
+    previewDraft: true
```
Comment (stuartromanek) [inline modules/@apostrophecms/doc-type/index.js:10]: unrelated, gets the Preview Draft button on Page create

File: modules/@apostrophecms/admin-bar/ui/apos/components/TheAposContextTitle.vue (hunk 1 above)
Comment (abea) [reply modules/@apostrophecms/admin-bar/ui/apos/components/TheAposContextTitle.vue:137]: you and your unrelated fiddling.

File: modules/@apostrophecms/ui/ui/apos/components/AposCellLabels.vue [hunk 3]
```diff
@@ -1,25 +1,36 @@
...
-    <span v-if="manuallyPublished && item.modified">
+    <span v-if="manuallyPublished && item.modified && !item.submitted && item.lastPublishedAt">
       <AposLabel
-        label="Active Draft" class="apos-table__cell-field__label"
+        label="Pending Updates" class="apos-table__cell-field__label"
+        tooltip="There are active changes to this document."
+        :modifiers="[ 'is-success', 'is-filled' ]"
+      />
+    </span>
+    <span v-if="item.submitted">
```
Comment (abea) [inline modules/@apostrophecms/ui/ui/apos/components/AposCellLabels.vue:13]: Seems like it should be a `v-else-if` since they're related.

**Review by abea — CHANGES_REQUESTED** [review]

File: modules/@apostrophecms/ui/ui/apos/components/AposCellLabels.vue (hunk 3 above)
Comment (stuartromanek) [reply modules/@apostrophecms/ui/ui/apos/components/AposCellLabels.vue]: aye

**Review by abea — APPROVED** [review]

**Review by abea — APPROVED** [review]

**Pull request merged**